com.unity.multiplayer.mlapi/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -1,8 +1,10 @@
 using System.Collections.Generic;
+#if MULTIPLAYER_TOOLS
```

Comment:
Not vetoing this or anything but I do wonder if it would be easier to just compile out the whole file. I don't think an IDE would correctly put a new using into this block if a new one is added. We would probably catch this in code review though